lore/urls.py

```diff
@@ -20,10 +20,13 @@
 from django.contrib import admin
 
 import importer.urls as importer_urls
+from learningresources.views import welcome, create_repo
 import cas.urls as cas_urls
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^importer/', include(importer_urls)),
+    url(r'^welcome/', welcome, name="welcome"),
```

Comment:
Can we please use an include here for URLs? If we still want to use the root url space, you can keep these two and redirect them to the app, or just have them be vanity URLs, but learningresources should have a urls.py of its own with its views.